crates/router/src/connector/bluesnap/transformers.rs

```diff
@@ -813,19 +813,75 @@ pub struct BluesnapWebhookBody {
 #[serde(rename_all = "camelCase")]
 pub struct BluesnapWebhookObjectEventType {
     pub transaction_type: BluesnapWebhookEvents,
+    pub cb_status: Option<BluesnapChargebackStatus>,
 }
 
 #[derive(Debug, Deserialize)]
 #[serde(rename_all = "SCREAMING_SNAKE_CASE")]
+pub enum BluesnapChargebackStatus {
+    #[serde(alias = "New")]
```

Comment:
Are we getting BluesnapChargebackStatus as SCREAMING_SNAKE_CASE as well as what you have defined in the alias?

Reply:
Yes we are getting both cases, depending on the Webhook event type, i.e `CHARGEBACK` or `CHARGEBACK_STATUS_CHANGE`